docs/dev/stream/testing.md

```diff
@@ -181,7 +181,7 @@ public class ExampleIntegrationTest extends StreamingMultipleProgramsTestBase {
 
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
-class ExampleIntegrationTest extends StreamingMultipleProgramsTestBase {
+class ExampleIntegrationTest extends ScalaStreamingMultipleProgramsTestBase {
```

Comment:
I see that in the code we still have only 1 usage of `ScalaStreamingMultipleProgramsTestBase`, but I don't see why it can't be replaced with `AbstractTestBase` also (therefore removing `ScalaStreamingMultipleProgramsTestBase` and not referencing it anymore in docs).

@zentol could you comment here?

Reply:
yes i agree, let's modify the docs to mention `AbstractTestBase` and remove `ScalaStreamingMultipleProgramsTestBase` in a separate issue.